masterlist.yaml

```
@@ -6854,6 +6854,17 @@ plugins:
     tag: [ Delev ]
 
 ###### Gameplay - Crafting & Harvesting ######
+  - name: 'BasicHarvest(Core|For).*\.esp'
```

The regex can be made a bit less greedy:

```suggestion
  - name: 'BasicHarvest(Core|For.*)\.esp'
```